docs/src/main/asciidoc/secretmanager.adoc

```diff
@@ -51,6 +51,8 @@ Spring Cloud GCP Secret Manager offers several configuration properties to custo
 | Name | Description | Required | Default value
 | `spring.cloud.gcp.secretmanager.bootstrap.enabled` | Enables loading secrets from Secret Manager as a bootstrap property source. Set this to **true** to enable the feature. | No | `false`
 | `spring.cloud.gcp.secretmanager.secret-name-prefix` | A prefix string to prepend to the property names of secrets read from Secret Manager | No | "" (empty string)
+| `spring.cloud.gcp.secretmanager.version` | Defines a specific version of secrets to read from Secret Manager | No | "" (empty string)
+| `spring.cloud.gcp.secretmanager.versions.<secret-id>` | Defines a version for a specific secret-id to read from Secret Manager | No | "" (empty string)
```

Comment:
Should probably mention that it takes precedence over `s.c.g.secretmanager.version`.